File: Sources/gestion-culturelle-front/project/src/app/model/animateurDto.ts

import { RoleDto } from './roleDto';
import { AnimationDto } from './animationDto';

export class AnimateurDto {
    id: number;
    nom: string;
    prenom: string;
    email: string;
    entreprise:string;    
    password: string;
    adresse: string;
    role: RoleDto;
    animation:AnimationDto;
    
  


    constructor(
        id?: number,
        nom?: string,
        prenom?: string,
        email?: string,
        password?: string,
        animation?: AnimationDto,
        adresse?:string,
        entreprise?:string,
        role?:RoleDto,      
        ){
        this.id = id;
        this.nom=nom;
        this.prenom=prenom;
        this.email=email;
        this.password=password;
        this.adresse=adresse;
        this.role=role;
        this.animation=animation;    
        this.entreprise=entreprise;        
       
    }
}

File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/AnimateurDto.java
package com.afpa.cda.dto;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class AnimateurDto  {

	private int id;
	private String nom;
	private String prenom;
	private String email;	
	private String password;
	private String tokenSecret;
	private String adresse;
	private RoleDto role;

	private String entreprise;
	private List <AnimationDto> animations;
	private String photo;

}
